### DISCUSSION TITLE
Why is the cacheDirectory changed in Stacks?

### DISCUSSION THREAD

**Discussion by silvenon** [remix-run/remix#4296] [Q&A] [answered]
I noticed that Stacks are designed so that every tool caches in `node_modules/.cache`, e.g. Remix is configured to cache in `node_modules/.cache/remix`. What is the reason for this?

**Comment by sergiodxa** [comment, marked as answer]
Why cache inside node_modules? So the developers have that already ignored by git

**Reply by silvenon** [reply]
Thanks for the answer! I think I understand, but why wouldn't that be the default already? Also, what mislead me is that the ESLint script being run on CI is configured to cache linting, so I thought that this cache is somehow being used on CI, but `node_modules` aren't being cached, so I guess not?

**Reply by kiliman** [reply]
I wonder if it has something to do with the Docker builds used by the stacks. Perhaps they found that adding the .cache in node_modules resulted in smaller images. 🤷‍♂️

**Reply by silvenon** [reply]
Makes sense!
